internal/controllers/topology/cluster/reconcile_state.go

```diff
@@ -727,3 +727,25 @@ func (r *Reconciler) reconcileReferencedTemplate(ctx context.Context, in reconci
 
 	return nil
 }
+
+// createErrorWithoutObjectName removes the name of the object from the error message. As each new Create call involves and
+// object with a unique generated name each error appears to be a different error. As the errors are being surfaced in a condition
+// on the Cluster, the name is removed here to prevent each creation error from triggering a new reconciliation.
+func createErrorWithoutObjectName(err error, obj client.Object) error {
+	out := ""
+	var statusError *apierrors.StatusError
+	if errors.As(err, &statusError) {
+		if statusError.Status().Details != nil {
+			for _, cause := range statusError.Status().Details.Causes {
+				out = fmt.Sprintf("%s %s: %s: %s", out, cause.Type, cause.Field, cause.Message)
```

Comment:
nit, looks like the first "%s" leads to a double space at the beginning. Can we maybe add those errors to a string array and then use  `strings.Join(x, " ")`?